app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,24 @@
+class TasksController < ApplicationController
+  def daily
+    # Hardcode first child for Phase 1
+    @child = Child.first || Child.create!(name: "Test Child")
+    @date = params[:date] ? Date.parse(params[:date]) : Date.current
+    @tasks = @child.active_tasks.ordered
```

Comment:
We should just have ordered as the default scope so here we can call @child.active_tasks. 

Add the scope to the child model with a simple unit test.